### PR TITLE
Mkdoc Documentation + Docker Init

### PR DESCRIPTION
**Pull request by jordan-rash**

### CODE REVIEW

**Comment by jordan-rash** [comment]
CC: @claymcleod @adthrasher

This PR takes the documentation that was already in place, and adds to to a GitHub Pages workflow utilizing Mkdocs.  Additionally, this PR has the initial Dockerfile with workflow to generate packages once along with releases.  Please send any and all feedback.  Mkdocs output can be previewed here: https://jordan-rash.github.io/RNAIndel

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        uses: docker/setup-qemu-action@v1
+      - 
+        name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - 
+        name: Login to GitHub Container Registry
+        uses: docker/login-action@v1 
+        with:
+          registry: ghcr.io
+          username: ${{ secrets.CLOUDY_USERNAME }}
```
Comment (jordan-rash) [inline .github/workflows/publish.yml:31]: @claymcleod This will need to be set in the repo settings

**Comment by claymcleod** [comment]
@rawagiha are you interested in reviewing this before me merge in?

**Review by rawagiha — APPROVED** [review]

File: .github/workflows/publish.yml (hunk 1 above)
Comment (claymcleod) [reply .github/workflows/publish.yml:31]: @jordan-rash both secrets have been added and @stjudecloud-cloudy should have access to push that docker image. Give it a shot.

**Review by claymcleod — APPROVED** [review]
LGTM, let's try and see how it goes.

File: .github/workflows/publish.yml (hunk 1 above)
Comment (jordan-rash) [reply .github/workflows/publish.yml:31]: @claymcleod I don't have access to merge on this repo. 
I will be be standby to make sure it all works.

**Comment by claymcleod** [comment]
@jordan-rash Okay, I merged it in.

**Pull request merged**